### PR TITLE
Add --local-aws-creds to create-efs-volume

### PR DESCRIPTION
**Pull request by jsafrane**
In STS mode, a cluster does not have usable AWS credentials. Add `--local-aws-creds` cmdline parameter, so local env. vars or `~/.aws/config` can be used instead.

Also add log messages around to see what's going on.

### CODE REVIEW

**Comment by jsafrane** [comment]
cc @openshift/storage

File: pkg/efscreate/efs.go [hunk 1]
```diff
@@ -62,32 +62,41 @@ func NewEFSSession(infra *v1.Infrastructure, sess *session.Session) *EFS {
...
+
+	klog.V(4).Info("Creating MountTargets")
 	mts, err := efs.createMountTargets()
 	if err != nil {
 		return "", err
 	}
 	efs.resources.mountTargets = mts
 
-	// wait for all mountTargets associated with filesystem ID to be become available
+	klog.V(4).Info("Waiting for MountTargets to get available")
```
Comment (bertinatto) [inline pkg/efscreate/efs.go:99]: IMO this could be safely logged as V(2)

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**